setup.py

```diff
@@ -86,15 +86,34 @@ def is_platform_mac():
     if not _CYTHON_INSTALLED:
         raise ImportError('No supported version of Cython installed.')
     from Cython.Distutils import build_ext as _build_ext
+    from Cython import Tempita as tempita
```

Comment:
i can find description in changelog, but 0.19.1 source includes ``Tempita`` as subpackage at least.

- https://github.com/cython/cython/blob/master/CHANGES.rst